mazerunner-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/ResumedSessionScenario.kt

```diff
@@ -13,33 +13,42 @@ import android.util.Log
  */
 internal class ResumedSessionScenario(config: Configuration,
                                       context: Context) : Scenario(config, context) {
+
+    companion object {
+        private const val SLEEP_MS: Long = 100
+    }
+
+    init {
+        config.autoCaptureSessions = false
+    }
+
     override fun run() {
         super.run()
         val client = Bugsnag.getClient()
         val thread = HandlerThread("HandlerThread")
         thread.start()
 
-        Handler(thread.looper).post {
+        Handler(thread.looper).post(Runnable {
```

Comment:
Fixes a lint warning about implicit SAM interfaces - this is functionally equivalent